### PR TITLE
Contact me form added

### PR DESCRIPTION
**Pull request by misikira**
1. has an html and css to style the contact form
2. connected tp form sphere so users can leave message

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,19 +208,16 @@ <h5 class="frameworks">frameworks </h5>
...
-</body>
+                <section class="form-contact-me">
+                    <h3>Contact me here</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Odit obcaecati id voluptatibus, architecto veniam reprehenderit! Est architecto consequuntur eum quo facilis cumque, nisi quia accusamus quidem quisquam facere, at fugit modi excepturi libero eos rerum. Aliquid pariatur inventore aperiam! Eius eligendi distinctio ipsum temporibus sunt!</p>
+                        <form action="https://formspree.io/f/xlekbypb"
+                        method="POST">
+                        <input type="text" name="name" placeholder="misikir teka" maxlength="30" required>
+                        <input type="email" name="email" placeholder="abc@abc.com" required>
+                        <textarea name="message" placeholder="Write your message here..." maxlength="500" required></textarea>
+                        <button class="btn" type="submit">Get in touch</button>
```
Comment (christianbiring1) [inline index.html:219]: - [ ] For a better look and feel Kindly make sure you stick to the original design as much as you can. the spacing, and size of elements are supposed to be almost the same as in the provided template.

### Your design :point_down: 
image
### Figme design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @misikira , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] The from spree service has been used correctly :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -208,19 +208,17 @@ <h5 class="frameworks">frameworks </h5>
...
-                
-               
-</body>
+                <section class="form-contact-me">
+                    <h3>Contact me here</h3>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                        I’d love to help with it!</p>
+                        <form action="https://formspree.io/f/xlekbypb"
+                        method="POST">
+                        <input type="text" name="name" placeholder="misikir teka" maxlength="30" required>
```
Comment (adel-gu) [inline index.html:217]: - [ OPTIONAL] Please consider adding a meaningful placeholder for name input, I highly suggested using `Full Name` or `Enter Your Full Name`

File: index.html [hunk 3]
```diff
@@ -208,19 +208,17 @@ <h5 class="frameworks">frameworks </h5>
...
+                        <input type="email" name="email" placeholder="abc@abc.com" required>
```
Comment (adel-gu) [inline index.html:218]: - [ OPTIONAL] Please consider adding a meaningful placeholder for email input, I highly suggested using `Email` or `Enter Your Email`

**Review by adel-gu — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @misikira 👋🏼 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- The contact layout is implemented ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**